js/index.js

```diff
@@ -84,6 +137,25 @@ function showProject(listId) {
 }
 
 
+function showProjectList() {
```

Comment:
Projects list that need to be displayed should be passed in the function argument.